task/hiffy/src/lpc55.rs

```diff
@@ -75,6 +75,8 @@ pub enum Functions {
     ReadFromSp((u32, u32), drv_sp_ctrl_api::SpCtrlError),
     #[cfg(feature = "spctrl")]
     SpCtrlInit((), drv_sp_ctrl_api::SpCtrlError),
+    #[cfg(feature = "spctrl")]
+    MeasureSp((), drv_sp_ctrl_api::SpCtrlError),
```

Comment:
It's confusing that the function is `db_reset_sp` but this is `MeasureSp`, maybe rename both sides to `ResetAndMeasureSp`?

Reply:
The measurement is a side-effect and won't happen if there is a powered STLINK attached to the SP. I can put comments in to that effect.